packages/react-sdk/src/components/rich-text-link.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import { RichTextLink as LinkPrimitive } from "./rich-text-link";
```

Comment:
shouldn't both link and richtextlink use the same underlying Link component?

Reply:
Idk, just made it consistent. Wanna reuse?

Reply:
I'll figure out this in https://github.com/webstudio-is/webstudio-designer/pull/488